src/components/PeoplePage.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect, useState } from 'react';
+import '../App.scss';
+import { getPeople } from '../api';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState([]);
+
+  useEffect(() => {
+    getPeople().then(setPeople);
```

Comment:
Instead of passing `people` array with props you could fetch them from server directly in the `PeopleTable` component